core/i18n/resources/en.js

```diff
@@ -727,19 +731,25 @@ Do you want to leave the page?`,
   },
 
   emails: {
+    signature: `<p>Thank you,<br>
+      The OpenForis Arena team
+      </p>`,
     userInvite: {
       subject: 'You have been invited to OpenForis Arena!',
       body: `<p>Hello,</p>
              <p>You have been invited to join the survey <strong>{{surveyLabel}}</strong> as {{groupLabel}}</p>
              {{temporaryPasswordMsg}}
              <p><a href="{{serverUrl}}">Click here to access OpenForis Arena</a></p>
-             <p>
-             Thank you,
-             <br>
-             The OpenForis Arena team
-             </p>`,
+             $t(emails.signature)`,
       temporaryPasswordMsg:
         '<p>Your username is <strong>{{email}}</strong> and your temporary password is <strong>{{password}}</strong></p>',
     },
+    userForgotPassword: {
+      subject: 'OpenForis Arena: password forgot',
+      body: `<p>Hello,</p>
+             <p>You have asked a new password to access the system.</p>
+             <p><a href="{{url}}">Click here to reset your password</a></p>
+             $t(emails.signature)`,
```

Comment:
I would say:

```
Hello {{name}},

Someone requested that the password for your OpenForis Arena account be reset.

<a href="{{url}}">Click here to reset your password</a>

If you didn't request this, you can ignore this email or let us know. Your password won't change until you create a new password. 

{{signature}}
```

Reply:
Also we can add `This link will expire in 48 hours.`

Reply:
yes